kivy/animation.py

```diff
@@ -437,24 +412,67 @@ def stop_property(self, widget, prop):
     def cancel(self, widget):
         self.anim1.cancel(widget)
         self.anim2.cancel(widget)
-        super(Sequence, self).cancel(widget)
+        super(CompoundAnimation, self).cancel(widget)
 
     def cancel_property(self, widget, prop):
         '''Even if an animation is running, remove a property. It will not be
         animated further. If it was the only/last property being animated,
         the animation will be canceled (see :attr:`cancel`)
 
         This method overrides `:class:kivy.animation.Animation`'s
-        version, to cancel it on all animations of the Sequence.
+        version, to cancel it on all animations of the CompoundAnimation.
 
-        .. versionadded:: 1.10.0
+        .. versionadded:: 1.11.0
         '''
         self.anim1.cancel_property(widget, prop)
         self.anim2.cancel_property(widget, prop)
         if (not self.anim1.have_properties_to_animate(widget) and
                 not self.anim2.have_properties_to_animate(widget)):
             self.cancel(widget)
 
+    def have_properties_to_animate(self, widget):
+        return (self.anim1.have_properties_to_animate(widget) or
+                self.anim2.have_properties_to_animate(widget))
+
+    @property
+    def transition(self):
+        # This property is impossible to implement
+        raise NotImplementedError()
```

Comment:
Yep I agree with `adding a message to the exception`.
And about AttributeError, I can't decide because for me both looks fine. But according to the [reference](https://docs.python.org/3/library/exceptions.html#NotImplementedError), AttributeError might be better.

Reply:
Because `NotImplementedError` means `real implementation required`, which is not our case.
so yeah AttibuteError is better I think.